COMPATIBILITY.md

```diff
@@ -0,0 +1,80 @@
+# Compatibility Guarantee
+## Introduction
+
+The release of InfluxDB version 1 is a major milestone in the
+development of the software lifecycle. InfluxDB 1 is a stable platform
+for the growth of programs and projects that utilize InfluxDB as a
+database.
+
+## Expectations
+
+Although we expect that the vast majority of users will maintain this
+compatibility over time, it is impossible to guarantee that no future
+change will break any program. This document is an attempt to set
+expectations for the compatibility of InfluxDB 1 in the future. There
+are a number of ways in which a client that works today may fail to do
+so after a future point release. They are unlikely but worth recording.
+
+* Security. A security issue in the specification or implementation may
+  come to light whose resolution requires breaking compatibility. We
+  reserve the right to address such security issues.
+* Unspecified behavior. The InfluxDB specification tries to be explicit
```

Comment:
Should we explicitly define the "specification" as referring to the InfluxQL, the HTTP API, and line protocol specs? Do we have formal HTTP API and line protocol specs?